### ISSUE TITLE
Introduce buttons for "View as"

### ISSUE THREAD

**Issue by IanMayo** [serge-web/serge#446]
When logged in as Umpire, we need a "View As" toolbar, to allow the umpire to double-check information is being displayed correctly for the player forces:
![image](https://user-images.githubusercontent.com/1108513/84660484-d2407a80-af10-11ea-8ab2-3f67836ee503.png)

Ian is happy to implement the processing logic, but doesn't know how to add the buttons.

It would be satisfactory for the buttons to fire a callback in the `mapping` component, like this:
```
const viewAsCallback = (force: string) => {
    console.log("view as clicked, force:", force)
}
```
 
So this issue is just to create the buttons, and to call the callback. One button will be created for each force that either has assets, or is listed in `Controlled By` for another force (white can control green in the current mock document).  Ian will implement the other logic.

**Comment by IanMayo** [comment]
Ian now has to process the new `View as` events.

**Comment by IanMayo** [comment]
Closed in #452
